index.js

```diff
@@ -162,9 +162,12 @@ function getMetrics({ jobId, startTime, endTime }) {
             return metrics;
         })
         .catch((err) => {
-            // if cannot get coverage, do not throw err
-            // eslint-disable-next-line max-len
-            logger.error(`Failed to get coverage and tests percentage for job ${jobId}: ${err.message}`);
+            // if there is no coverage measurement target, 404 is returned and this is not an error
+            if (err.statusCode !== 404) {
```

Comment:
Does all 404 error is something which no need to catch? It might better to throw it away by the message of error.

Reply:
fixed.
If we get a 404 error for an unexpected reason that the ID doesn't exist, we probably get a 404 error when send a put request to sonar, so I think that's enough. However, it's better that the difference in behavior is smaller, so I accepted the proposal.

The 404 error that sonar actually returns when there is no id is shown below.
```
{
  name: 'StatusCodeError',
  statusCode: 404,
  message: `404 - {"errors":[{"msg":"Component key 'job:1' not found"}]}`,
  (snip)
}
```